lib/src/media/image.dart

```diff
@@ -118,8 +118,6 @@ class _ViamRGBADecoder extends img.Decoder {
 
   @override
   img.Image? decodeFrame(int frame) {
-    Uint8List imageData;
```

Comment:
[q] why remove this here? is it a _driveby_ that you noticed was unused?

Reply:
Yup, unused